app/src/sharedTest/java/org/oppia/app/parser/HtmlParserTest.kt

```diff
@@ -128,14 +128,35 @@ class HtmlParserTest {
     onView(withId(R.id.test_html_content_text_view)).check(matches(not(textView.text.toString())))
   }
 
+  @Test
+  fun testHtmlContent_customSpan_isAdded() {
+    val textView = activityTestRule.activity.findViewById(R.id.test_html_content_text_view) as TextView
+    val htmlParser = htmlParserFactory.create(/* entityType= */ "", /* entityId= */ "", /* imageCenterAlign= */ true)
+    val htmlResult: Spannable = htmlParser.parseOppiaHtml(
+      "<p>You should know the following before going on:<br></p>" +
+          "<ul><li>The counting numbers (1, 2, 3, 4, 5 ….)<br></li>" +
+          "<li>How to tell whether one counting number is bigger or smaller than another<br></li></ul>",
+      textView
+    )
+
+    /* Reference: https://medium.com/androiddevelopers/spantastic-text-styling-with-spans-17b0c16b4568#e345 */
+    val bulletSpans = htmlResult.getSpans<Any>(0, htmlResult.length, Any::class.java)
```

Comment:
Shouldn't we be checking how many CustomBulletSpans are present?

Reply:
Changed `Any` to `CustomBulletSpan` to fix this.